docs/framework/20sources/30implementation/mixing-control-data-flow.md

```diff
@@ -1,15 +1,46 @@
-### Mixing of Control and Data Flow
+### Mixing of Control and Data Plane
+
+The terms *Control Plane* and *Data Plane* come from networking [[1](https://en.wikipedia.org/wiki/Control_plane)]. In the
```

Comment:
nit: the reference number is static and likely to get out of sync, suggest removing and associating the link to a relevant phrase.

Reply:
Good point. I replaced with the source - Wikipedia.